Documentation/Contributors/CodingGuide/README.md

```diff
@@ -135,6 +136,12 @@ function Model(options) {
 
 * Text files, including JavaScript files, end with a newline to minimize the noise in diffs.
 
+## Code Guidelines
+
+For syntax and style guidelines, we adhere to the common JavaScript "recommended" rules, according to [ESLint](http://eslint.org/docs/rules/). Some
```

Comment:
Instead of adhere, I would say something like "We use the ESLint recommend setting as a base and extend it with additional rules.  No need to mention specific rules since we will link to the various files.